lib/sdf-server/src/service/v2/func/create_unlocked_copy.rs

```diff
@@ -54,6 +55,11 @@ pub async fn create_unlocked_copy(
     let code = get_code_response(&ctx, new_func.id).await?;
     let summary = new_func.into_frontend_type(&ctx).await?;
 
+    let variant = if let Some(schema_variant_id) = request.schema_variant_id {
+        SchemaVariant::get_by_id(&ctx, schema_variant_id).await?
+    } else {
+        None
+    };
```

Comment:
Aren't all funcs attached to assets these days? How is this possible? (unrelated to your PR, just curious if you know)

Reply:
Funcs can technically be attached to only components, it's in a bit of a half state though as we don't expose it in the UI today.